'Replace duplicate `addLine` statement with SSRVControlPointGenerator

chinsyo opened the pull request.

chinsyo committed:
Make it better

chinsyo committed:
bump pod version number

chinsyo committed:
Merge pull request #1 from Jerrrr/master

update to latest version.

chinsyo committed:
Replace duplicate `addLine` statement with SSRVControlPointGenerator

ApolloZhu commented on Source/SwiftyStarRatingView.swift:
```diff
@@ -8,6 +8,7 @@
 
 import UIKit
 
+public typealias SSRVControlPointGenerator = (CGFloat, CGFloat) -> CGPoint
```
Does this need to be `public`?

chinsyo replied:
Maybe not, unless we'll add `custom star shape path` feature.

chinsyo committed:
Update to private

chinsyo commented:
@ApolloZhu Update to private, request for review.

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 106.80kb -> 79.64kb (25.43%)

/Example/Example/Assets.xcassets/3.imageset/3.png -- 13.77kb -> 5.56kb (59.59%)
/Example/Example/Assets.xcassets/2.imageset/f123.png -- 13.30kb -> 8.06kb (39.43%)
/Example/Example/Assets.xcassets/1.imageset/f006.png -- 14.95kb -> 9.32kb (37.68%)
/Image/sx.png -- 64.78kb -> 56.71kb (12.46%)

chinsyo committed:
Merge pull request #2 from chinsyo/imgbot

[ImgBot] Optimize images

chinsyo committed:
update swift version to 5.0
